src/Shared/UnitTests/EngineTestEnvironment.cs

```diff
@@ -235,7 +235,9 @@ private IEnumerable<(ILogger logger, Func<string> textGetter)> GetLoggers()
                     foreach (var pair in pairs)
                     {
                         var expectedText = pair.expected.textGetter();
+                        expectedText = expectedText.Substring(expectedText.IndexOf("Project"));
```

Comment:
All these extra substrings in tests are because the text log still has environment variables at the top, and the binlog is now hiding that.